File: samples/js/upload-audio-link.js
const API_KEY = "";     // API key here
const AUDIO_LINK = "";  // Audio link here

var axios = require('axios');
var data = JSON.stringify({"link": AUDIO_LINK});

var config = {
  method: 'post',
  url: 'https://api-us.musiio.com/api/v1/upload/audio-link',
  auth: {
      username: API_KEY,
      password: ""
  },
  headers: { 
    'Content-Type': 'application/json'
  },
  data : data
};

axios(config)
.then(function (response) {
  console.log(JSON.stringify(response.data));
})
.catch(function (error) {
  console.log(error);
});

File: samples/python-individual/upload-audio-link.py
import requests
from requests.auth import HTTPBasicAuth
import json

api_key = ""    # Your API Key
audio_link = "" # Your Audio link

url = "https://api-us.musiio.com/api/v1/upload/audio-link"

payload = {
    "link": audio_link
}

headers = {
  'Content-Type': 'application/json'
}

response = requests.post(url, headers=headers, data=json.dumps(payload), auth=HTTPBasicAuth(api_key, ''))

print(response.text.encode('utf8'))